tests/test_nginx.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+set -x
+
+WORKPATH=$(dirname "$PWD")
+LOG_PATH="$WORKPATH/tests"
+ip_address=$(hostname -I | awk '{print $1}')
+
+function build_docker_images() {
+    cd $WORKPATH/comps/nginx/docker
+    docker build --no-cache --build-arg https_proxy=$https_proxy --build-arg http_proxy=$http_proxy -t opea/nginx:comps -f ./Dockerfile .
+    if [ $? -ne 0 ]; then
+        echo "opea/nginx built fail"
+        exit 1
+    else
+        echo "opea/nginx built successful"
+    fi
+}
+
+function start_service() {
+    export NGINX_PORT=80
+
+    # Start Docker Containers
+    docker run -d --name opea-xeon-nginx-server -p 80:80 opea/nginx:comps
```

Comment:
name the container as `test-comps-*`, so it could be removed by workflow even the test failed.

Reply:
ok, updated